### ISSUE TITLE
Loco not running when run on loader unloader

### ISSUE THREAD

**Issue by iceman11a** [CovertJaguar/Railcraft#466]
The loco seems to stop at all loaders and unloaders.Even when the loaders or unloaders are not setup for that loco. I thought it mite have been some thing else. So I started a new world and tried it and I get the same result. 

If I have a unloader setup for a fluid cart and for water. The loco stops and hangs there. I tried pushing it and I just can't get it to go. I applied a lever to it and that seem to release the loco. I wonder why I ever had this problem before. ?

http://youtu.be/0EiYD0VoIdQ

**Comment by Forecaster** [comment]
You need to provide the Railcraft version you are using and preferably a list of any other mods you have.

**Comment by CovertJaguar** [comment]
I don't see this happening in my test world.

**Comment by Forecaster** [comment]
SkySom (on irc) was unable to verify as well.

**Comment by CovertJaguar** [comment]
This does sound like the same problem that Kris1432 was having though.

**Comment by iceman11a** [comment]
I'm using the DNS Modpack. The newest version of the Mod pack and railcraft 9.4.0.0

**Comment by CovertJaguar** [comment]
You'll need to test it more explicitly. Start by trying only Railcraft, absolutely nothing else, not even NEI.

From there you need start testing with various combinations of mods. Binary Search methods can greatly simplify this processes.

**Comment by iceman11a** [comment]
I have been testing it in 3 worlds. I all so fount a problem with the rec box. From what I fount out that both of them don't seem to send out a red stone signal. How ever I will keep testing.

**Comment by iceman11a** [comment]
CJ, Sorry I did an updated video. I won't test any more unless your willing to pay me for it. Every one else wants me to pay them. It's only fair. It looks like a red stone signal issue. With the money your making. Sorry I won't do any more testing. 

http://youtu.be/qbFJHd8L4Xg

**Comment by CovertJaguar** [comment]
@iceman11a I can't duplicate it with my setup, I tried.

**Comment by iceman11a** [comment]
Well I been chatting with Kris Brown, and he says you know about his problem and it does match the same problem I have. If you can't duplicate it. and if you want me too. I can send you this SP world. The one in the video. and see if your having the same problem. It could be some thing else. Make sure you make a video on what you do while your on this world. Let me know and i can upload it and give you the link. See I have a online world and a big one. I would hate to have to trash that world.

**Comment by CovertJaguar** [comment]
Its not a world I need, its for someone to pick apart an installation where this happens and find the specific combination of mods that causes it.

You might try giving "[Railcraft]" op permissions if you are using some kind of permissions plugin.

**Comment by iceman11a** [comment]
I did find out that it seems to be related to the mod pack I use. I have another setup on a localhost with only 99 mods where the modpack I all so use has 190 mods. If the DNS Modpack does have a plugin for some kind of permissions plugin. I don't know any thing about it. and I do play in creative mode.

**Comment by iceman11a** [comment]
I wanted to let you know that the problem only seems to be in this mod pack for some reason. I don't have this problem any where else. I just tried updating the modpack and I still have the same problem. I'll have to check to see if any one else is having this same problem other the Kris and I.

**Comment by LoPhatKao** [comment]
ive tried several combos since i saw this video, cannot duplicate with any of these versions:
mc-1.7.10
forge- 1230,1284,1291
bc- 6.18,6.26,6.31,6.33
rc- 9.4

as CJ guessed and you later confirmed, it is a different mod in your pack interfering
you need to get the modpack maker to look into it, since you have stated that you will not

**Comment by iceman11a** [comment]
LoPhatKao, Thanks for the info. and Yes I did. I did report this. and never got a reply on there IRC chat,. SOme one by the name of Kris Brown came up with some ideas to fix this and some of them do work. I will be making another video showing ever one a fix. How ever I'm still testing and playing with it. I will have to contact the DNS techpack team and see if any one else has this same problem other the Kris and I.
